tasks/pkg-debian.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - name: Install apt-transport-https
   apt: name=apt-transport-https state=present
+  when: not ansible_check_mode
```

Comment:
why is this needed? Shouldn't the package be always in the apt repo?

Reply:
if no ansible job using apt was ever run, `--check` will fail because `python-apt` is not install.

```
fatal: [127.0.0.1]: FAILED! => {"changed": false, "msg": "python-apt must be installed to use check mode. If run normally this module can auto-install it."}
```

Adding the check allow users to check their playbook without installing python-apt.